components/map/basic/src/leaflet/marker-manager.js

```diff
@@ -141,8 +141,12 @@ class MarkerManager {
   }
 
   // Coupled FC, should be removed in the future
-  isVisited() {
-    return false
+  isVisited({propertyInfo}) {
```

Comment:
Quizá se puede reducir a:

```suggestion
  isVisited({propertyInfo}) => !!propertyInfo && !!propertyInfo.isVisited
```

Reply:
Para que mantenga compatibilidad lo que comenta Rubén, debería ser así:

```suggestion
  isVisited({propertyInfo} = {}) => !!propertyInfo && !!propertyInfo.isVisited
```

Reply:
You miss equal symbol 👍 

```javascript
isVisited = ({propertyInfo} = {}) =>
    !!propertyInfo && !!propertyInfo.isVisited
```

I will commit changes now.